homeassistant/components/qbittorrent/sensor.py

```diff
@@ -38,13 +57,15 @@
         vol.Required(CONF_USERNAME): cv.string,
         vol.Required(CONF_PASSWORD): cv.string,
         vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+        vol.Optional(CONF_MONITORED_CONDITIONS, default=DEFAULT_CONDITIONS): vol.All(
+            cv.ensure_list, [vol.In(SENSOR_TYPES)]
+        ),
```

Comment:
Adding this goes against [ADR-0003](https://github.com/home-assistant/architecture/blob/master/adr/0003-monitor-condition-and-data-selectors.md):

> We don't merge PRs with data selectors. We remove existing unneeded selectors until Home Assistant 1.0. We focus on a good view/frontend to select and group available data.

Reply:
Ah OK I hadn't seen that, I saw [here](https://developers.home-assistant.io/docs/creating_platform_code_review/#0-common) about the use of MONITORED_CONDITIONS and assumed it was OK as didn't see any mention there of any issues regarding using.

Can you make any suggestions as to what to change? Sorry if it's a basic question, this is already way over my head!

Reply:
Yeah, monitored conditions is still used in some integrations. Ideally you would add a config flow so users can set it up via the frontend and do not have to alter their YAML. With a config flow a user can also easily disable sensors that they do not want to show on the frontend.

However, in this case I think it would be okay to just remove the monitored conditions and just set up all of them automatically.

Reply:
Hi springstan, I've added everythingsmarthomes to my PR and extended the qbittorrent integration with flow and added some services https://github.com/home-assistant/core/pull/43661